interfaces/coin-sig.repl

```diff
@@ -0,0 +1,54 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Using formal verification in Pact interfaces to create example coin.
+
+(begin-tx)
+
+(define-keyset "test-keyset" (sig-keyset))
```

Comment:
We can move this to GOVERNANCE with a dummy gov function, cuts down on keyset noise